src/utils/buildCssOverridesStyle.ts

```diff
@@ -0,0 +1,17 @@
+export type CssVariableOverrides = {
+  [key: string]: string;
+};
+
+export const buildCssOverridesStyle = (
+  base: string,
```

Comment:
If you make this last it can be optional so a user doesn't need to pass empty string if they don't have other overrides.